jest.setup.js

```diff
@@ -0,0 +1,7 @@
+/**
+ * In the tests, performance.now sometimes does not change as fast as Date.now.
+ * This results in short intervals sometimes having a duration of 0ms and flaky tests.
+ */
+global.performance.now = () => {
+    return Date.now();
+};
```

Comment:
In our experience `Date.now()` produce inconsistent value (at least at runtime), are we sure the same thing won't happen here?

Reply:
This is probably a change in the way `performance` is mocked by jest https://jestjs.io/docs/jest-object#jestusefaketimersfaketimersconfig. 
The mock became less accurate - I did not dig deep here, I am not sure it is worth it.